plugins/mcreator-core/variables/actionresulttype.json

```diff
@@ -1,5 +1,5 @@
 {
-  "color": 90,
+  "color": "90",

```

Comment:
Doesn't int value get serialized into String properly too? Maybe try using a lenient parser option when parsing if it does not work out of the box. This way we can keep existing plugins working with numbers

Reply:
It actually does, I thought it wouldn't